gap/oper.gi

```diff
@@ -1725,6 +1725,38 @@ function(D, list)
   return DigraphShortestDistance(D, list[1], list[2]);
 end);
 
+InstallMethod(VerticesReachableFrom, "for a digraph and a vertex",
+[IsDigraph, IsPosInt],
+function(D, root)
+  local N, index, current, succ, visited, prev, n, i, parent;
+  N := DigraphNrVertices(D);
+  index := ListWithIdenticalEntries(N, 1);
+  current := root;
+  succ := OutNeighbours(D);
+  visited := [];
```

Comment:
```suggestion
  visited := [root];
```

Reply:
I'm not 100% if you want the root in the list or not, so please take or leave this suggestion as appropriate

Reply:
I've argued above that the root should not be in the returned if and only if (as documented) it is reachable from itself by a **non-trivial** walk. That explained partly why the code was structured as it was.

Reply:
This was my reasoning to @marinaanagno https://github.com/digraphs/Digraphs/pull/379/files#r553562550

Reply:
Yup, makes sense to me too, looks to me like @marinaanagno used this definition